addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/LutronHandler.java

```diff
@@ -46,7 +46,7 @@ private void sendCommand(LutronCommand command) {
         IPBridgeHandler bridgeHandler = getBridgeHandler();
 
         if (bridgeHandler == null) {
-            this.logger.info("Not sending command, no bridge associated");
+            logger.info("Not sending command, no bridge associated");
```

Comment:
Remove this logging statement. The thing status change below already has this information.